### PR TITLE
Show problem

### PR DESCRIPTION
**Pull request by Changron**
can show problems in contest

### CODE REVIEW

**Comment by nickelcheng** [comment]
Please merge the latest code to fix db schema typo.

**Comment by Changron** [comment]
merged and fix pager

**Comment by nickelcheng** [comment]
Why do your layout different with origin layout (in problem)
yours:
screenshot from 2015-04-13 20 59 12
origin:
screenshot from 2015-04-13 21 00 55

File: contest/templates/contest/contest.html [hunk 1]
```diff
@@ -111,22 +111,35 @@ <h1 class="panel-title" style="text-align:center;">Clarification</h1>
     <!--Contest detail-->
     <div id="detail">
       <ul class="nav nav-tabs" role="tablist" id="contest_tab">
-        <li class="active"><a href="#overview" role="tab" data-toggle="tab">Overview</a>
+        <li class="active" role="presentation"><a href="#overview" role="tab" data-toggle="tab">Overview</a>
```
Comment (nickelcheng) [inline contest/templates/contest/contest.html]: This line is too long

File: contest/templates/contest/contest.html [hunk 2]
```diff
@@ -111,22 +111,35 @@ <h1 class="panel-title" style="text-align:center;">Clarification</h1>
...
         </li>
-        <li><a href="#problem" role="tab" data-toggle="tab">Problem</a>
+        <li class="dropdown" role="presentation">
+          <a href="#problem" class="dropdown-toggle" data-toggle="dropdown">
+            Problem<span class="caret"></span>
```
Comment (nickelcheng) [inline contest/templates/contest/contest.html]: There is nothing in ```span``` tag, is it ok to remove it?

Comment (Changron) [inline contest/templates/contest/contest.html]: the tag itself is a icon
http://getbootstrap.com/components/#glyphicons-examples

**Comment by Changron** [comment]
I simply include the problem page.
BTW, It looks all the same here.

File: contest/templates/contest/contest.html (hunk 2 above)
Comment (nickelcheng) [inline contest/templates/contest/contest.html]: ok

**Comment by nickelcheng** [comment]
Can you add current page and total page for pager? Just like this:
screenshot from 2015-04-13 21 09 56

**Comment by Changron** [comment]
fixed

**Comment by nickelcheng** [comment]
+1, going to merge

**Pull request merged**